test/features/showandhide/basicFields.feature

```diff
@@ -0,0 +1,39 @@
+@functional @showandhide
+Feature: Hide and show of basic fields
+
+  Background:
+    Given I have logged in
+
+  Scenario Outline: fields on the same page visible when show & hide condition is met based on mandatory and optional fields
+    Given a case type containing conditionals exists
+    And I start createCase event
+    When I enter '<value>' into the text field with id '<fieldId>'
+    Then the field with label '<expectedLabel>' is visible
+
+    Examples:
+      | value            | fieldId           | expectedLabel          |
+      | showmethemoney   | TextField         | Text Field 2           |
+      | showme           | TextFieldOptional | Text Field ShowHide 13 |
+
+  Scenario Outline: fields on the same page NOT visible when show & hide condition is NOT met based on mandatory and optional fields
+    Given a case type containing conditionals exists
+    And I start createCase event
```

Comment:
we should verify the fields are initially visible

Reply:
In this case the field is initially invisible. It is because the showCondition is set to hide it when its value is not set to 'showmethemoney'.